src/display/api.js

```diff
@@ -472,6 +479,8 @@ function _fetchDocument(worker, source, pdfDataRangeTransport, docId) {
       isEvalSupported: source.isEvalSupported,
       fontExtraProperties: source.fontExtraProperties,
       enableXfa: source.enableXfa,
+      standardFontDataUrl: source.standardFontDataUrl,
+      standardFontDataWorkerFetch: source.standardFontDataWorkerFetch,
```

Comment:
Given that the JSDocs for the `standardFontDataUrl` options says "Include the trailing slash." it thus cannot be empty and I think it should suffice to only send *one* parameter to the worker:
```javascript
standardFontDataUrl:
  source.standardFontDataWorkerFetch ? source.standardFontDataUrl : null,
```